epigraphhub/data/foph.py

```diff
@@ -1,28 +1,33 @@
 """
-The functions in this module allow the user to get the datasets stored in the 
-epigraphhub database. 
+The functions in this module allow the user to get the datasets stored in the
+epigraphhub database.
 
-The function get_agg_data aggregate the data according to the values of one column, 
-and the method of aggregation applied. 
+The function get_agg_data aggregate the data according to the values of one column,
+and the method of aggregation applied.
 
-The function get_georegion_data filter the datasets for a list of 
+The function get_georegion_data filter the datasets for a list of
 selected regions (in the Switzerland case: cantons).
 
 The function get_cluster_data is focused on being used to apply the forecast models.
-This function returns a table where each column is related to a  different table and 
-region (e.g. the daily number of cases and the daily number of hospitalizations in 
-Geneva and Fribourg). 
+This function returns a table where each column is related to a  different table and
+region (e.g. the daily number of cases and the daily number of hospitalizations in
+Geneva and Fribourg).
 Some parts of the code of this function are focused on the swiss case.
```

Comment:
In the upcoming refactoring, we will move country-specific code to their own modules